packages/controllers/src/admin.rs

```diff
@@ -1,5 +1,6 @@
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
+use std::fmt;
```

Comment:
Nit: Personally I would add whole `std::fmt::Debug` here and second file..

Reply:
I think I use this other places as well.
Let's change them all in another PR if this seems more clear. (feel free to make an issue)